gen-cpp/CppServer.cpp

```diff
@@ -0,0 +1,51 @@
+#include <thrift/protocol/TBinaryProtocol.h>
+#include <thrift/server/TServer.h>
```

Comment:
use include order like this:

#include <files from your library>
...
<empty line>
#include <files from the standard library>
...
<empty line>
#inlcude <files from the other libraries>
...